### PR TITLE
Feature/front page phone form

### PR DESCRIPTION
**Pull request by palmfjord**
Moving logic from download page -> own component and add component to landing hero.
screenshot 2018-11-14 at 17 53 41

Initially just putting it up here to get a pr instance, will ask for a review later.

### CODE REVIEW

File: src/components/AppLinkForm.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    };
+    // utmParams take precendent over default branch params
+    const linkOptions = utmParamsToBranchLinkOptions(
+      utmParams,
+      defaultBranchLinkOptions,
+    );
+    (window as any).branch.sendSMS(
+      phoneNumber,
+      {
+        ...linkOptions,
```
Comment (hedvigoscar) [inline src/components/AppLinkForm.tsx]: This should help us identify people who use this form down the road.

```suggestion
        ...linkOptions,
        is_desktop_splash_form: true,
```

Comment (palmfjord) [reply src/components/AppLinkForm.tsx]: we should add this as a prop then, because it's used on ie `/download` too

Comment (hedvigoscar) [reply src/components/AppLinkForm.tsx]: There was no direct need to do it before as there was only one SMS form location. But yeah, we can add this property there too for sure.

**Comment by palmfjord** [comment]
@frapperino @hedvigoscar u wanna take a final look? merge ETA 5 minutes

**Pull request merged**